pkg/operator/ceph/object/spec.go

```diff
@@ -431,6 +452,29 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) (v1.Container,
 			container.Args = append(container.Args, c.sseKMSVaultTLSOptions(kmsEnabled)...)
 		}
 	}
+	if c.store.Spec.Gateway.OpsLogSidecar != nil {
+		container.Env = append(container.Env, podNameEnvVars...)
+		container.Args = append(container.Args,
+			cephconfig.NewFlag("rgw_enable_ops_log", "true"),
+			cephconfig.NewFlag("rgw_ops_log_file_path", opsLogAbsFilename),
+		)
+	} else {
+		// older version of Rook was using mon db to store rgw ops log config params.
+		// cleanup previous value in case if ops log was enabled in older version:
```

Comment:
```suggestion
		// older version of Rook was using mon db to store rgw ops log config params.
		// cleanup previous value in case if ops log was enabled in older version:
		// TODO: only needed in Rook v1.17. remove after
```

I don't think the TODO comment is urgently necessary, but it might be good to note that this is only needed for 1.17. As Travis suggested, we can probably merge and backport this PR and then immediately follow up to remove this else block in master (v1.18). I lean toward including some kind of comment just in case something urgent comes up and follow up is delayed/forgotten

Reply:
done